flink-libraries/flink-table/src/test/java/org/apache/flink/api/java/batch/table/CastingITCase.java

```diff
@@ -138,50 +138,5 @@ public void testCastFromString() throws Exception {
 		String expected = "1,1,1,1,2.0,2.0,true\n";
 		compareResultAsText(results, expected);
 	}
-
-	@Ignore // Date type not supported yet
-	@Test
-	public void testCastDateFromString() throws Exception {
```

Comment:
Yes, they have been moved to `TimeTypesTest`. This is just a forgotten test that is not necessary anymore.